app/models/rspec-test/spec/user_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../../user'
+
+RSpec.describe User do
+  describe '#id' do
+    context 'idがnilでない場合' do
+      it 'present?がtrue' do
```

Comment:
IMO

個人的には、文章として言い切った方がテストとして読みやすいかもと思った。

```suggestion
      it 'present?がtrueになる' do
```